tests/integration/test_select.py

```diff
@@ -126,7 +124,7 @@ def test_timezones(c, datetime_table):
     "input_table",
     [
         "long_table",
-        pytest.param("gpu_long_table", marks=pytest.mark.gpu),
+        # pytest.param("gpu_long_table", marks=pytest.mark.gpu),
```

Comment:
is the gpu test failing in this case?

Reply:
hmm forgot this was commented out. Let me uncomment and see after the next commit.